src/routes/legislative-process/+page.svelte

```diff
@@ -1 +1,86 @@
-<h1>ร่างกฎหมายกลายเป็นกฎหมายได้อย่างไร</h1>
+<script land="ts">
+	import dayjs from 'dayjs';
+	import buddhistEra from 'dayjs/plugin/buddhistEra';
+	import 'dayjs/locale/th';
+
+	dayjs.extend(buddhistEra);
+	dayjs.locale('th');
+</script>
+
+<div class="flex flex-col w-full">
+	<header class="bg-teal-20">
+		<div class="w-full max-w-[800px] px-10 py-10 md:py-20">
+			<h1 class="fluid-heading-05 mb-4">ร่างกฎหมายกลายเป็นกฎหมายได้อย่างไร</h1>
+			<p>
+				<span class="text-text-primary font-bold">อัปเดตล่าสุด :</span>
+				<span class="text-helper-text-01">{dayjs(new Date()).format('DD MMM BBBB')}</span>
+			</p>
+		</div>
+	</header>
+	<div class="w-full max-w-[800px] flex flex-col gap-10 px-4 md:px-10 py-8 bg-ui-01 md:bg-white">
+		<section class="flex flex-col gap-4 bg-white px-4 md:px-0 py-8 md:py-0">
```

Comment:
You might notice that we use the same class for every section, h1, div, 52, hr. To clean this up, I think we have 2 choices.

1. We should be able to use Tailwind's `@apply` together with Svelte's `style` tag to add the same class to every element as follows:

```svelte
<style>
section {
  @apply flex flex-col gap-4 bg-white px-4 md:px-0 py-8 md:py-0;
}
/* other elements */
</style>
```
2. Split those duplicated codes into a new component (can be created in `/components/legislative-process`), which receives props or a slot to render the content.

Reply:
Agreed! I might working on the first choice to clean up.